emscripten.py

```diff
@@ -312,19 +310,26 @@ def update_settings_glue(settings, metadata):
   # Integrate info from backend
   if settings['SIDE_MODULE']:
```

Comment:
Thereby raising the question, should we include these settings changes in the wasm_backend path?